mayhem/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM alpine:3.13 as builder
```

Comment:
Perhaps you could add your own goaccess for fuzzing?

Reply:
Sorry, what do you mean by my "own goaccess"?